src/Components/Button.lua

```diff
@@ -77,6 +77,7 @@ end
 
 function Button:render()
 	local flat = self.props.Flat
+	local action = self.props.Action
```

Comment:
This should be a more descriptive name like `onClicked` or something similar. It also doesn't really need to be a local variable here; it can be moved to the event connection.